include/ddc/detail/kokkos.hpp

```diff
@@ -13,6 +13,16 @@
 
 #include "macros.hpp"
 
+#if defined(MDSPAN_USE_BRACKET_OPERATOR)
+#if MDSPAN_USE_BRACKET_OPERATOR
+#define DDC_MDSPAN_ACCESS_OP(mds, ...) mds[__VA_ARGS__]
+#else
+#define DDC_MDSPAN_ACCESS_OP(mds, ...) mds(__VA_ARGS__)
+#endif
+#else
+#define DDC_MDSPAN_ACCESS_OP(mds, ...) mds[__VA_ARGS__]
+#endif
+
```

Comment:
I am a bit confused by this logic. 
In case a variable `MDSPAN_USE_BRACKET_OPERATOR` is not defined, why we will use a bracket operator?

Reply:
I got it.
Could you please comment on this in the code.

Reply:
Yes I will update